src/storybook/helpers/with-gallery-decorator.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Decorator } from '@storybook/react'
+import { Gallery } from '../../gallery'
+import { GalleryProps } from '../../types'
+
+export const withGalleryDecorator =
+  (galleryProps: GalleryProps = {}): Decorator =>
+  (StoryFn) => {
+    return (
+      <Gallery {...galleryProps}>
+        <StoryFn />
+      </Gallery>
+    )
+  }
```

Comment:
Previously, we did not have such decorator, because it was an idea, that our stories - examples of `react-photoswipe-gallery` usage.

These examples should be as simple as possible - users should be able to copy them and paste in their project.
If we have some decorator - it can be a little bit harder for new users to understand how to use our library.

So, if we still want out stories to be simple-to-copy and simple-to-understand examples, we should remove this decorator.